elodie/elodie.py

```diff
@@ -11,23 +11,23 @@
 
```

Comment:
What do you think about also symlinking it back to the original location? I imagine people have some automated scripts pointing to the current location which may fail and be difficult to troubleshoot if they simply do an update of the code.

Git seems to handle symlinks well enough. https://stackoverflow.com/a/954575

Reply:
It would be better to go the other way around - move the file and then create a symlink from the original location to `elodie/elodie.py`